server/container/guice/blob-export-guice/src/main/java/org/apache/james/modules/LocalFileBlobExportMechanismModule.java

```diff
@@ -37,6 +37,10 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(LocalFileBlobExportMechanismModule.class);
 
+    @Override
+    protected void configure() {
```

Comment:
I bet we can remove that empty method no?

Reply:
no :) if I don't override it, then guice will use the configure() method at parent class, then I will get an very weird exception `java.lang.AbtractMethodError` . I can show you that error

Reply:
I took me hours